src/go/rpk/pkg/cli/registry/schema/check_compatibility.go

```diff
@@ -84,6 +93,7 @@ func newCheckCompatibilityCommand(fs afero.Fs, p *config.Params) *cobra.Command
 	cmd.Flags().StringVar(&schemaType, "type", "", fmt.Sprintf("Schema type (%v); overrides schema file extension", strings.Join(supportedTypes, ",")))
 	cmd.Flags().StringVar(&sversion, "schema-version", "", "Schema version to check compatibility with (latest, 0, 1...)")
 	cmd.Flags().StringVar(&refs, "references", "", "Comma-separated list of references (name:subject:version) or path to reference file")
+	cmd.Flags().BoolVarP(&verboseCompat, "verbose-compat", "", false, "If the schema fails the compatibility test, output the reason(s)")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&verboseCompat, "verbose-compat", "", false, "If the schema is not compatible, output the reasons")
```

Reply:
I think we want verbose compatibility by default. In what case would we not want the extra information?

Reply:
When it's not asked for with `?verbose=true`?

Reply:
Fair point. So maybe a more appropriate flag here would be `--quiet` or something like that. `--brief`? idk.

Reply:
But, we _always_ want to print a good error. The problem we have (here and in Console) is that we just outright _can't_. This is similar to how in the past we would just print INVALID_TOPIC_EXCEPTION with standard Kafka wording when failing to create a topic: we eventually improved the ErrorMessage from Redpanda, and now we actually print _why_ topic creation failed. I think that here, we can avoid a flag entirely and always print a detailed error message.

Reply:
All makes sense. Presence of the flag on the API is primarily for compatibility w/ the reference impl, but I'm happy to defer to you all for CLI behavior (default or otherwise). 

Constructing the messages on the broker side isn't exactly free, but I haven't been able to contrive a scenario where it would be a real issue on its own.